app/assets/stylesheets/profile.scss

```diff
@@ -0,0 +1,38 @@
+h3 {
+  color: white;
+  text-align: center;
+  margin-top: 15%;
+}
+
+a {
+  color: white;
+}
+
+a:hover {
+  color: #FF851B;
+}
+
+.joined-opp {
+  border: 1px solid #FF851B;
+  padding: 5px;
+}
+
+.user-info {
+  text-align: center;
+}
+
+.joined-opp, .user-info, .orgs-list {
+  margin-top: 10%;
+  padding: 1%;
+  border: 1px solid #FF851B;
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`